docs/reference/benchmarks.md

```diff
@@ -12,31 +12,33 @@ This page describes each benchmark that is part of fuzzbench.
-| harfbuzz-1.3.2              | fuzz-target            | False           | 58                    | 10021                    | 6.24                 |
-| jsoncpp_jsoncpp_fuzzer      | jsoncpp_fuzzer         | True            | 0                     | 5536                     | 5.96                 |
-| lcms-2017-03-21             | fuzz-target            | False           | 1                     | 6959                     | 6.11                 |
-| libjpeg-turbo-07-2017       | fuzz-target            | False           | 1                     | 9586                     | 6.4                  |
-| libpcap_fuzz_both           | fuzz_both              | False           | 0                     | 8149                     | 6.14                 |
-| libpng-1.2.56               | fuzz-target            | False           | 1                     | 2991                     | 5.8                  |
-| libxml2-v2.9.2              | fuzz-target            | False           | 0                     | 50461                    | 8.23                 |
-| mbedtls_fuzz_dtlsclient     | fuzz_dtlsclient        | False           | 1                     | 10942                    | 6.67                 |
-| openssl_x509                | x509                   | True            | 2241                  | 45989                    | 18.14                |
-| openthread-2019-12-23       | fuzz-target            | False           | 0                     | 17932                    | 6.72                 |
-| php_php-fuzz-parser         | php-fuzz-parser        | True            | 2782                  | 123767                   | 15.57                |
-| proj4-2017-08-14            | fuzz-target            | False           | 44                    | 6156                     | 6.17                 |
-| re2-2014-12-09              | fuzz-target            | False           | 0                     | 6547                     | 6.02                 |
-| sqlite3_ossfuzz             | ossfuzz                | True            | 1258                  | 45136                    | 7.9                  |
-| systemd_fuzz-link-parser    | fuzz-link-parser       | False           | 6                     | 53453                    | 5.91                 |
-| vorbis-2017-12-11           | fuzz-target            | False           | 1                     | 5022                     | 6.0                  |
-| woff2-2016-05-06            | fuzz-target            | False           | 62                    | 10923                    | 6.72                 |
-| zlib_zlib_uncompress_fuzzer | zlib_uncompress_fuzzer | False           | 0                     | 875                      | 5.69                 |
+| Benchmark                   | Fuzz target            | Has dictionary? | Seed type\*              | Number of seed inputs  | Number of progam edges\*\*| Binary Size (MB)\*\*\*|
```

Comment:
Would you be ok to fix the "Has dictionary column", i see more 

```
jsoncpp_jsoncpp_fuzzer/build.sh:cp $SRC/jsoncpp/src/test_lib_json/fuzz.dict $OUT/jsoncpp_fuzzer.dict
lcms-2017-03-21/Dockerfile:RUN wget -qO $OUT/fuzz-target.dict \
lcms-2017-03-21/Dockerfile:    https://raw.githubusercontent.com/google/fuzzing/master/dictionaries/icc.dict
libpng-1.2.56/Dockerfile:RUN wget -qO $OUT/fuzz-target.dict \
libpng-1.2.56/Dockerfile:    https://raw.githubusercontent.com/google/fuzzing/master/dictionaries/png.dict
libxml2-v2.9.2/Dockerfile:RUN wget -qO $OUT/fuzz-target.dict \
libxml2-v2.9.2/Dockerfile:    https://raw.githubusercontent.com/google/AFL/debe27037b9444bbf090a0ffbd5d24889bb887ae/dictionaries/xml.dict
openssl_x509/build.sh:cp fuzz/oids.txt $OUT/x509.dict
php_php-fuzz-parser/build.sh:cp sapi/fuzzer/dict/parser $OUT/php-fuzz-parser.dict
proj4-2017-08-14/Dockerfile:RUN wget -qO $OUT/fuzz-target.dict \
proj4-2017-08-14/Dockerfile:    https://raw.githubusercontent.com/google/fuzzing/master/dictionaries/proj4.dict
re2-2014-12-09/Dockerfile:RUN wget -qO $OUT/fuzz-target.dict \
re2-2014-12-09/Dockerfile:    https://raw.githubusercontent.com/google/fuzzing/master/dictionaries/regexp.dict
sqlite3_ossfuzz/Dockerfile:COPY build.sh *.dict $SRC/
sqlite3_ossfuzz/build.sh:cp $SRC/*.dict $SRC/*.zip $OUT/
```

Reply:
hadnt touched it but ok ... ;)

Reply:
:) got you!